scss/boost_union/post.scss

```diff
@@ -1185,68 +1190,195 @@ body.theme-boost-union-commincourse {
-            margin-top: 1.5rem;
-            width: $blockregionoutsiderightwidth;
+/* And on larger screens and below. */
+@include media-breakpoint-down(md) {
+    /* Center the main region content while maintaining full width for the left and right block regions,
+    thereby positioning the page content in the middle of the page. */
+    body.limitedwidth {
+        .left-region-nextmaincontent {
+            &.main-inner-outside-left-right.main-inner-outside-nextmaincontent,
+            &.main-inner-outside-left.main-inner-outside-nextmaincontent,
+            &.main-inner-outside-right.main-inner-outside-nextmaincontent,
+            &.main-inner-outside-left-right.main-inner-outside-nearwindowedges,
+            &.main-inner-outside-left.main-inner-outside-nearwindowedges,
+            &.main-inner-outside-right.main-inner-outside-nearwindowedges {
+                display: grid;
+                #theme-block-region-outside-left {
+                    order: 1;
+                }
+                #theme-block-region-outside-right {
+                    order: 2;
+                }
+            }
         }
+    }
+}
 
-        /* If the admin configured outside blocks to be shown next to the main content. */
-        .main-inner-outside-nextmaincontent {
-            #theme-block-region-outside-left,
-            #theme-block-region-outside-right {
-                /* Disable float (just to be sure). */
-                float: none;
+/* On even larger screens. */
+@include media-breakpoint-up(lg) {
+    /* Set the width and margin of the outside-left region. */
+    #theme-block-region-outside-left {
+        margin-right: 1rem;
+        margin-top: 1.5rem;
+        width: $blockregionoutsideleftwidth;
+    }
+
+    /* Set the width of the outside-right region. */
+    #theme-block-region-outside-right {
+        margin-left: 1rem;
+        margin-top: 1.5rem;
+        width: $blockregionoutsiderightwidth;
+    }
+
+    /* Set the width of the main region content to 50%, allowing the left and right block regions to have a wider width */
+    body.limitedwidth #page.drawers .main-inner-wrapper {
+        &.main-inner-outside-left-right,
+        &.main-inner-outside-left,
+        &.main-inner-outside-right {
+            /* If the admin configured outside blocks to be shown next to the window edges. */
+            &.main-inner-outside-nearwindowedges {
+                /* Center the main region content to position the page content in the middle of the page,
+                and place the left and right block regions near the window edges. */
+                display: flex;
             }
         }
-
-        /* If the admin configured outside blocks to be shown next to the main content. */
-        .main-inner-outside-nearwindowedges {
-            /* Float the outside-left block region to the left. */
-            #theme-block-region-outside-left {
-                float: left;
+        .main-inner {
+            flex: auto;
```

Comment:
I think this statement should be moved inside the

```
       &.main-inner-outside-left-right,
        &.main-inner-outside-left,
        &.main-inner-outside-right {}

```

bracket to make sure it only applies if block regions are enabled and to keep the default behaviour if not.